lib/braspag-rest/hashie/iutrash.rb

```diff
@@ -0,0 +1,16 @@
+module Hashie
+  class IUTrash < Hashie::IUDash
+    include Hashie::Extensions::Dash::PropertyTranslation
+
+    def inverse_attributes
+      {}.tap do |attributes|
+        self.class.translations.each do |(from, property)|
```

Comment:
No, before I'm using translations_hash that was a little bit different and needs that, but this translations is a simple hash... fixed 933c4dbf95b3bce3b8fc555bb578f01dc43a96c9